File: swagger/models/get_client_contracts_response.py
from .base_model import BaseModel

from .client_contract import ClientContract
from .pagination_response import PaginationResponse

class GetClientContractsResponse(BaseModel):
	"""
	no description available

	Attributes:

		contracts
		Contains the details of the client’s contract.

		pagination_response
		Contains information about the pagination used.
	"""

	input_map = {
		'Contracts': ['contracts', ClientContract],
		'PaginationResponse': ('pagination_response', PaginationResponse),
		}

	output_map = {
		'contracts': 'Contracts',
		'pagination_response': 'PaginationResponse',
		}

	contracts = None
	pagination_response = None

File: swagger/Models/GetClientContractsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClientContractsResponse extends BaseModel
{
	/**
	 * Contains the details of the client’s contract.
	 * 
	 * @var ClientContract[]
	 */
	public $contracts;
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;

	protected function getInputMap()
	{
		return [
			'Contracts' => ['contracts', ClientContract::class],
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'contracts' => 'Contracts',
			'paginationResponse' => 'PaginationResponse',
			];
	}
}
